[GHC-9.0.x] Library won't build with GHC 9.0.0

Library won't build with GHC 9.0.0, please see log snipped below...

```
[   35s] /usr/bin/ghc --make -fbuilding-cabal-package -O -static -dynamic-too -dynosuf dyn_o -dynhisuf dyn_hi -outputdir dist/build -odir dist/build -hidir dist/build -stubdir dist/build -i -idist/build -ilib -idist/build/autogen -idist/build/global-autogen -Idist/build/autogen -Idist/build/global-autogen -Idist/build -optP-include -optPdist/build/autogen/cabal_macros.h -this-unit-id fmt-0.6.1.2-5dfGrAr0oy5IbhTcMOmpY -hide-all-packages -Wmissing-home-modules -no-user-package-db -package-db dist/package.conf.inplace -package-id base-4.15.0.0 -package-id base64-bytestring-1.2.0.1-wMVwxdzWeX9EOWhIVqH1Y -package-id bytestring-0.10.12.0 -package-id call-stack-0.2.0-F3nLWnfeL064cC2St4eCDt -package-id containers-0.6.2.1 -package-id formatting-7.1.1-1BFIJ7rMS602NMTv9NRMBR -package-id microlens-0.4.12.0-IoK2ncpVKVzEwRUasmz5D3 -package-id text-1.2.4.0 -package-id time-1.9.3 -package-id time-locale-compat-0.1.1.5-6us7pZxstqD6EDxOHL8mgH -XHaskell2010 Fmt Fmt.Time Fmt.Internal Fmt.Internal.Core Fmt.Internal.Formatters Fmt.Internal.Template Fmt.Internal.Tuple Fmt.Internal.Numeric Fmt.Internal.Generic -Wall -fno-warn-unused-do-bind -Wno-redundant-constraints
[   35s] [1 of 9] Compiling Fmt.Internal.Core ( lib/Fmt/Internal/Core.hs, dist/build/Fmt/Internal/Core.o, dist/build/Fmt/Internal/Core.dyn_o )
[   36s] [2 of 9] Compiling Fmt.Internal.Formatters ( lib/Fmt/Internal/Formatters.hs, dist/build/Fmt/Internal/Formatters.o, dist/build/Fmt/Internal/Formatters.dyn_o )
[   36s] 
[   36s] lib/Fmt/Internal/Formatters.hs:201:57: error:
[   36s]     Ambiguous occurrence ‘singleton’
[   36s]     It could refer to
[   36s]        either ‘Data.List.singleton’,
[   36s]               imported from ‘Data.List’ at lib/Fmt/Internal/Formatters.hs:12:1-16
[   36s]               (and originally defined in ‘base-4.15.0.0:Data.OldList’)
[   36s]            or ‘Data.Text.Lazy.Builder.singleton’,
[   36s]               imported from ‘Data.Text.Lazy.Builder’ at lib/Fmt/Internal/Formatters.hs:25:1-49
[   36s]               (and originally defined in ‘Data.Text.Internal.Builder’)
[   36s]     |
[   36s] 201 |     spaces = mconcat $ replicate (T.length bullet + 1) (singleton ' ')
[   36s]     |                                                         ^^^^^^^^^
[   36s] error: Bad exit status from /var/tmp/rpm-tmp.OYC1Ku (%build)
[   36s] 
[   36s] 
[   36s] RPM build errors:
[   36s]     Bad exit status from /var/tmp/rpm-tmp.OYC1Ku (%build)
```

--- a/lib/Fmt/Internal/Formatters.hs
+++ b/lib/Fmt/Internal/Formatters.hs
@@ -9,7 +9,7 @@ module Fmt.Internal.Formatters where
 
 
 -- Generic useful things
-import Data.List
+import Data.List (intersperse)
 import Lens.Micro
 #if __GLASGOW_HASKELL__ < 804
 import Data.Monoid ((<>))

